examples/vl3_basic/helm/vl3/templates/vl3-nse-ucnf.tpl

```diff
@@ -97,3 +100,19 @@ data:
           prefixLength: 22
           routes: []
        ifName: "endpoint0"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: "nse-pod-service-{{ .Values.nsm.serviceName }}"
+  labels:
+    cnns/monitoring: vpp
+spec:
+  type: ClusterIP
+  selector:
+      cnns/nse.servicename: {{ .Values.nsm.serviceName | quote }}
+  ports:
+    - name: monitoring
+      port: 9191
```

Comment:
9191 is the port hardcoded in VPP. This is the port VPP is listening on. this will only change if VPP eventually decides to change its metrics port for whatever reason.